src/android/CodePush.java

```diff
@@ -242,6 +273,7 @@ private void returnStringPreference(String preferenceName, CallbackContext callb
     }
 
     private void handleAppStart() {
+        this.isRunningBinaryVersion = true;
```

Comment:
is `handleAppStart()` called multiple times? If once, do still need this given that you instantiated it to `true` above?